prow/cmd/deck/main.go

```diff
@@ -1037,7 +1038,11 @@ lensesLoop:
 	}
 
 	artifactsLink := ""
-	gcswebPrefix := cfg().Deck.Spyglass.GCSBrowserPrefixes.GetGCSBrowserPrefix(org, repo)
+	bucket := ""
+	if jobPath != "" && strings.HasPrefix(jobPath, providers.GS) {
```

Comment:
I was going to say this breaks when we use S3 but then again S3 doesn't work with gcsweb so seems fine